packages/beasties/src/index.js

```
@@ -500,9 +500,9 @@
       markOnly((rule) => {
         if (rule.type === 'comment') {
           // we might want to remove a leading ! on comment blocks
-          // critters can be part of "legal comments" which aren't striped on build
-          const crittersComment = rule.text.match(/^(?<!\! )critters:(.*)/);
-          const command = crittersComment && crittersComment[1];
+          // beasties can be part of "legal comments" which aren't striped on build
+          const beastiesComment = rule.text.match(/^(?<!\! )beasties:(.*)/);
+          const command = beastiesComment && beastiesComment[1];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)